pkg/utils/downloaders.go

```diff
@@ -193,6 +194,9 @@ func genEVELiveImage(image, outputDir string, format string, configDir string, s
 	if size == 0 {
 		dockerCommand = fmt.Sprintf("-f %s live", format)
 	}
+	if platform != "" && platform != "none" {
```

Comment:
I think we can use empty default value in cli or check for DefaultEVEPlatform here.